bundles/specmate-dbprovider-api/src/com/specmate/dbprovider/api/migration/IAttributeToSQLMapper.java

```diff
@@ -89,6 +89,15 @@ void migrateNewDoubleAttribute(String objectName, String attributeName, Double d
 	 * @throws SpecmateException
 	 */
 	void migrateNewObjectReference(String objectName, String attributeName) throws SpecmateException;
```

Comment:
what multiplicity is this migrationcode doing? please change the method name accordingly to be consistent